staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -74,6 +74,17 @@ var (
 	}
 )
 
+// The configuration objects we want to remove from the cluster.
+// This is a typed array instead of free form names, since the same name
+// can belong to objects of either type.
+var (
+	// The FlowSchema objects we want to remove
+	RemoveFlowSchemas = []*flowcontrol.FlowSchema{}
```

Comment:
Do we need these?

I was initially thinking that instead of defining them explicitly, every object that has defined "AutoUpdate" annotation and is not present in the bootstrap configuration should be removed.

Reply:
@wojtek-t Sure, if we want to use the `AutoUpdate` annotation key as a marker. So if a cluster operator creates a new flowschema/pl object by copy pasting from an existing bootstrap configuration with the annotation intact then we will be deleting those indiscriminately.

if we use the `AutoUpdate` marker approach, we would have to LIST all pl/flowschema for every polling interval (`1m` here). I think we can optimize by using a `lister` with cache so we can avoid making these LIST calls, thoughts?

the static approach is more efficient run-time but comes with a maintenance cost of making sure to add the removed object in their designated list. (which can be improved by adding a unit test that fails if a configuration is removed from the create list but not added to the designated removal list)

Reply:
> So if a cluster operator creates a new flowschema/pl object by copy pasting from an existing bootstrap configuration with the annotation intact then we will be deleting those indiscriminately.

Yes - that's what I would like to have.

> if we use the AutoUpdate marker approach, we would have to LIST all pl/flowschema for every polling interval (1m here). I think we can optimize by using a lister with cache so we can avoid making these LIST calls, thoughts?

I don't expect people having a lot of them. LISTing once a minute tens of objects sounds ok.
Yes - we should have a lister for that, but I wouldn't block this PR on it.